docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -78,18 +97,15 @@ To add a link:
    The link will appear listed under the Grafana News heading.
 1. To vote for a link, click the triangle icon next to the name of the link.
 
-## Log in to Grafana
+## Create a contact point for Grafana Managed Alerts
 
 Besides being an open-source observability tool, Grafana has its own built-in alerting service. This means that you can receive notifications whenever there is an event of interest in your data, and even see these events graphed in your visualizations.
 
-In your browser, navigate to [localhost:3000](http://localhost:3000).
-You should get logged in automatically
-
-## Create a contact point for Grafana Managed Alerts
-
 In this step, we'll set up a new contact point. This contact point will use the _webhooks_ channel. In order to make this work, we also need an endpoint for our webhook channel to receive the alert. We will use [requestbin.com](https://requestbin.com) to quickly set up that test endpoint. This way we can make sure that our alert is actually sending a notification somewhere.
 
-1. Browse to [requestbin.com](https://requestbin.com).
+1. In your browser, navigate to [localhost:3000](http://localhost:3000).
```

Comment:
I would add something like "navigate to [localhost:3000](http://localhost:3000), where Grafana is running"